### ISSUE TITLE
Resource: google_compute_target_https_proxy; apply/plan/refresh fails to update tfstate correctly

### ISSUE THREAD

**Issue by TrungLam** [terraform-providers/terraform-provider-google#244]
Hi guys,

So we're using terraform to create a target_https_proxy on GCP. It works and our infrastructure got built correctly, but fails to update tfstate file with ssl_certificate field. 

### Terraform Version
Terraform v0.9.11

### Affected Resource(s)
- google_compute_target_https_proxy

### Terraform Configuration Files
```hcl
resource "google_compute_target_https_proxy" "REDACT-https-proxy" {
  name = "REDACT-https-proxy"

  description = "https proxy"
  url_map = "${google_compute_url_map.REDACT-0-url-map.self_link }"

  ssl_certificates = ["https://www.googleapis.com/compute/beta/projects/REDACT/global/sslCertificates/REDACT-ssl"]
}
```

### Expected Behavior
Expected terraform plan to say that infrastructure is up to date

### Actual Behavior
The previous terraform apply did not update the tfstate file with ssl_certificates field

### Steps to Reproduce
1. `terraform apply`
2. `terraform plan`

**Comment by danawillow** [comment]
@rosbo I think the work you've been doing fixes this, right?

**Comment by rosbo** [comment]
I tested locally and I confirm that my fix #210 fixes this issue. It was released in version 0.1.2 (July 20, 2017) the **Terraform Google** provider.
